src/pages/SignUp/components/SignupForm.tsx

```diff
@@ -28,12 +26,25 @@ type Validated = {
 
 interface SignUpFormProps {
   setAuthToken: (auth: { token: string; username: string }) => void;
+  passwordLabel?: string;
+  passwordValue?: string;
+  onChangePassword?: (value: string, event: React.FormEvent<HTMLInputElement>) => void;
+  isValidPassword?: boolean;
+  isShowPasswordEnabled?: boolean;
+  showPasswordAriaLabel?: string;
+  hidePasswordAriaLabel?: string;
 };
 
```

Comment:
The only prop being passed to the `SignUpForm.tsx` is the `setAuthToken` right? This is where the implementation of the `SignupForm` is different to the `LoginForm`. I would probably only use that as a reference.

Reply:
I have passed all required props in SignUpForm.tsx. Then also I need to use only that as reference?

Reply:
If you look at the SignUp.tsx , these props are not being passed though.